README.md

```diff
@@ -6,62 +12,59 @@
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/1342f842a14f40cca856d6e81204f8ac)](https://app.codacy.com/gh/Sudokuru/Frontend/dashboard?utm_source=gh&utm_medium=referral&utm_content=&utm_campaign=Badge_grade)
 [![Maintainability](https://api.codeclimate.com/v1/badges/4cd1d1027b7b2532c680/maintainability)](https://codeclimate.com/github/Sudokuru/Frontend/maintainability)
 
-# Start the frontend app
-
-Run `npm i` in the root folder  
-Run `npm run start`
+## Sudokuru is an open-source project focused on developing a world-class, cross-platform Sudoku app. We aim to provide a delightful user experience while also contributing to the community by building a collection of reusable software modules. These modules are designed to be free, well-documented, modern, and interoperable, allowing developers to easily incorporate them into their own Sudoku-related projects.
 
-For testing with a mobile device you can download the Expo app.  
-You will also need to be logged in to Expo on the development device (laptop/desktop that is running the code).  
-To login type: `expo login` and fill out the parameters.
+### This frontend module is the primary user interface for Sudokuru, providing a cross-platform Sudoku experience built with React Native Web and TypeScript. It offers a clean, intuitive design with features such as:
 
-Contact Thomas to get added to the Expo organization so that you can log in with your own email and password.
+- 🎮 Play Sudoku across 9 different difficulty levels generated by the Sudokuru Clearinghouse module
+- 🎓 Learn how to play Sudoku with lessons from the basics all the way to advanced strategies
+- 📊 Statistics to track your progress
+- ⚙️ Various preferences including ones to customize dynamic board highlighting
+- 📅 _Upcoming_: custom strategy based hints from the Sudokuru npm library module
+- 📅 _Upcoming_: drills which let you practice individual strategies also powered by the Sudokuru npm library module
 
-For iOS, scan the QR code with your camera app, for Android you will need to scan the code from within the Expo app.
+# 🛠️ Local development
 
-For Web, hit the w key to start up the website at `localhost:19000`
+## ⚙️ General Setup:
 
-# Only use single puzzle for debugging
+1. Git clone this repository
```

Comment:
Might be better to replace with the git command:

1. Run `git clone https://github.com/Sudokuru/Frontend`

Reply:
Since some people use https and others use ssh urls for git clone going to keep it the way it is.